autoload/intero/process.vim

```diff
@@ -188,6 +188,7 @@ function! s:start_buffer(height) abort
     set bufhidden=hide
     set noswapfile
     set hidden
+    set filetype=intero
```

Comment:
Do we want to call this `filetype=intero`? `filetype=ghci`? `filetype=intero-neovim`?